pkg/quota/image/registry.go

```diff
@@ -12,11 +12,24 @@ import (
 // streams.
 func NewImageRegistry(osClient osclient.Interface) quota.Registry {
 	imageStream := NewImageStreamEvaluator(osClient)
+	return &generic.GenericRegistry{
+		InternalEvaluators: map[unversioned.GroupKind]quota.Evaluator{
+			imageStream.GroupKind(): imageStream,
+		},
+	}
+}
+
+// NewImageRegistry returns a registry for quota evaluation of OpenShift resources related to images and image
+// streams.
+func NewImageRegistryForAdmission(osClient osclient.Interface) quota.Registry {
```

Comment:
Apologize as I leave a trail of bread crumb comments on this PR, but this is my next step in understanding the need to split on admission control.  It's not yet clear from here why this split is needed.  A comment here would really help explain why.  Right now, its not-obvious to me why you don't collapse the admission registry into the normal `NewImageRegistry` call.  I need to continue my investigation.